README.md

```diff
@@ -150,18 +194,20 @@ For inspiration please take a look at the related
 
 For a new release, create a new branch. The branch name must be a semantic version
 number. Make sure you change the semantic version number and, for major releases,
-the codename in the README and config files (./configs/*)
+the codename in the README and config files (`./configs/`)
```

Comment:
drop the "./" prefix here, as you did everywhere in the document.

Reply:
by the way: currently you can directly reference "configs/common.config" for changing the codename.

Reply:
Done.